scripts/prepublish.js

```diff
@@ -0,0 +1,13 @@
+var sh = require('shelljs');
+
+sh.exec('eslint src test', function(code, output) {
+
+  if(code === 0){
```

Comment:
Nit pick: missing space before `{` and unneeded newlines before and after `if` block.